### ISSUE TITLE
NTR: VLCAD enzyme in fibroblasts low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9101]
New term request VLCAD enzyme in fibroblasts low (segal_071121171040): Very long chain acyl CoA dehydrogenase

**Comment by pnrobinson** [comment]
@MickeySegal
suggested term

Label: Reduced very long-chain acyl-CoA dehydrogenase activity
Def: Diminished activity of Very long-chain acyl-CoA dehydrogenase (VLCAD), an enzyme that catalyzes the initial step of mitochondrial beta-oxidation of long-chain fatty acids with a chain length of 14 to 20 carbons. VLCAD activity can be measured with biochemical testing using cultured fibroblasts or lymphocytes.
PMID:20301763

Please let me know if the location in fibroblasts needs to be represented in the term.

**Comment by MickeySegal** [comment]
As with others, location in fibroblasts need not be specified in your term.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000860
- label: Diminished tissue very long-chain acyl-CoA dehydrogenase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:20301763
- def: Diminished activity of Very long-chain acyl-CoA dehydrogenase (VLCAD), an enzyme that catalyzes the initial step of mitochondrial beta-oxidation of long-chain fatty acids with a chain length of 14 to 20 carbons. VLCAD activity can be measured with biochemical testing using cultured fibroblasts or lymphocytes.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9101
- synonyms:
